README.md

```diff
@@ -33,7 +33,7 @@ The very latest version is always available via 'github' by invoking one of the
 
 ```
 ## For the traditional ssh-based Git interaction:
-$ git clone git://github.com/JohnLangford/vowpal_wabbit.git
+$ git clone git://github.com:JohnLangford/vowpal_wabbit.git
```

Comment:
@andyreagan  - Can you undo this line change?
(trying this on Ubuntu) It breaks the cloning process from perfectly working to a failure:

```
tmp$ git clone git://github.com:JohnLangford/vowpal_wabbit.git
Cloning into 'vowpal_wabbit'...
fatal: unable to connect to github.com:JohnLangford:
github.com:JohnLangford: Name or service not known
```